frontend/src/components/SignIn/SignIn.css

```diff
@@ -1,116 +1,21 @@
-:root {
-  --input-padding-x: .75rem;
-  --input-padding-y: .75rem;
-}
-
-html,
-body {
-  height: 100%;
-}
-
-body {
-  display: -ms-flexbox;
-  display: -webkit-box;
-  display: flex;
-  -ms-flex-align: center;
-  -ms-flex-pack: center;
-  -webkit-box-align: center;
-  align-items: center;
-  -webkit-box-pack: center;
-  justify-content: center;
-  padding-top: 40px;
-  padding-bottom: 40px;
-  background-color: #f5f5f5;
-}
-
-.form-signin {
-  width: 100%;
-  max-width: 400px;
-  padding: 15px;
-  margin: 0 auto;
-}
 
-.form-label-group {
+.signInPassword.form-group.row {
   position: relative;
-  margin-bottom: 1rem;
-}
-
-.form-label-group>input,
-.form-label-group>label {
-  padding: var(--input-padding-y) var(--input-padding-x);
-}
-
-.form-label-group>label {
-  position: absolute;
-  top: 0;
-  left: 0;
-  display: block;
-  width: 100%;
-  margin-bottom: 0;
-  /* Override default `<label>` margin */
-  line-height: 1.5;
-  color: #495057;
-  border: 1px solid transparent;
-  border-radius: .25rem;
-  transition: all .1s ease-in-out;
-}
-
-.form-label-group input::-webkit-input-placeholder {
-  color: transparent;
-}
-
-.form-label-group input:-ms-input-placeholder {
-  color: transparent;
+  margin-bottom: 1%;
 }
 
-.form-label-group input::-ms-input-placeholder {
-  color: transparent;
+.forgotPassword.form-group.row > a {
+  color: grey;
```

Comment:
We have some particular greys that the designer gave us in the Style Guide. Maybe #9b9b9b or #696969 here?